src/Data/Unit/Polymorphic.agda

```diff
@@ -0,0 +1,38 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- The universe polymorphic unit type and the total relation on unit
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Unit.Polymorphic where
+
+open import Level
+open import Relation.Nullary
+open import Relation.Binary
+open import Relation.Binary.PropositionalEquality
+
+------------------------------------------------------------------------
+-- A unit type defined as a record type
+
+record ⊤ {ℓ : Level} : Set ℓ where
+  constructor tt
+
+------------------------------------------------------------------------
+-- An ordering relation over the unit type
+
+record _≤_ {ℓ ℓ′ e : Level} (x : ⊤ {ℓ}) (y : ⊤ {ℓ′}) : Set e where
+
+------------------------------------------------------------------------
+-- Decidable Equality and Ordering
```

Comment:
Decidability properties usually go in the `Properties` file. If we then put the remaining contents in the `Data.Unit.Polymorphic.Base` file, then `Data.Unit.Polymorphic` can export `Base` and the decidability proofs from `Properties`. I know it seems like overkill in this case, but I think it's best to be consistent on this...

Reply:
I knew I was taking a chance on using 2 files instead of the 3 that `Data.Unit` uses for this. I'll split it up.

Reply:
Done now.